src/main/java/com/zeepy/server/community/domain/Community.java

```diff
@@ -1,92 +1,109 @@
 @Getter
 @Entity
 public class Community {
-    @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "community_sequence_gen")
-    @SequenceGenerator(name = "community_sequence_gen", sequenceName = "community_sequence")
-    private Long id;    //id
-
-    @NotNull
-    @Enumerated(EnumType.STRING)
-    private CommunityCategory communityCategory;    //커뮤니티 카테고리
-
-    @Nullable
-    private String productName;
-
-    @Nullable
-    private Integer productPrice;
-
-    @Nullable
-    private String sharingMethod;
-
-    @Nullable
-    private Integer targetNumberOfPeople;
-
-    @Nullable
-    private Integer targetAmount;
-
-    @NotNull
-    private String title;
-
-    @NotNull
-    private String content;
-
-    @NotNull
-    @ManyToOne
-    @JoinColumn(name = "user_id")
-    private User user;
-
-    @OneToMany(mappedBy = "community")
-    private List<CommunityLike> likeUsers = new ArrayList<>();
-
-    @Deprecated
-    private String Comments;
-
-    @Deprecated
-    private String AchievementRate;
-
-    @OneToMany(mappedBy = "community")
-    private List<Participation> participationsList;
-
-    @ElementCollection
-    @JoinTable(name = "communityImageUrls", joinColumns = @JoinColumn(name = "communityID"))
-    private List<String> imageUrls;   //사진
-
-    @Builder
-    public Community(
-            CommunityCategory communityCategory,
-            String productName,
-            Integer productPrice,
-            String sharingMethod,
-            Integer targetNumberOfPeople,
-            Integer targetAmount,
-            String title,
-            String content,
-            List<String> imageUrls
-    ) {
-        this.communityCategory = communityCategory;
-        this.productName = productName;
-        this.productPrice = productPrice;
-        this.sharingMethod = sharingMethod;
-        this.targetNumberOfPeople = targetNumberOfPeople;
-        this.targetAmount = targetAmount;
-        this.title = title;
-        this.content = content;
-        this.imageUrls = imageUrls;
-    }
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "community_sequence_gen")
+	@SequenceGenerator(name = "community_sequence_gen", sequenceName = "community_sequence")
+	private Long id;    //id
+
+	@NotNull
+	@Enumerated(EnumType.STRING)
+	private CommunityCategory communityCategory;    //커뮤니티 카테고리
+
+	@Nullable
+	private String productName;
+
+	@Nullable
+	private Integer productPrice;
+
+	@Nullable
+	private String sharingMethod;
+
+	@Nullable
+	private Integer targetNumberOfPeople;
+
+	@Nullable
+	private Integer targetAmount;
+
+	@NotNull
+	private String title;
+
+	@NotNull
+	private String content;
+
+	@NotNull
+	@ManyToOne
+	@JoinColumn(name = "user_id")
+	private User user;
+
+	@OneToMany(mappedBy = "community")
+	private List<CommunityLike> likes = new ArrayList<>();
+
+	@Deprecated
+	private String Comments;
+
+	@Deprecated
+	private String AchievementRate;
```

Comment:
```suggestion
	@Deprecated
	private String comments;

	@Deprecated
	private String achievementRate;
```
변수명은 소문자로 시작합시당. Deprecated지만 그래도 신경써야겠죠?